app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/pixels/DeviceShieldPixelNames.kt

```diff
@@ -129,6 +130,7 @@ enum class DeviceShieldPixelNames(override val pixelName: String, val enqueue: B
 
     ATP_APP_BREAKAGE_REPORT("m_atp_breakage_report"),
     ATP_APP_BREAKAGE_REPORT_UNIQUE("m_atp_breakage_report_u"),
+    ATP_APP_CPU_MONITOR_REPORT(PixelPrefixes.ATP_APP_CPU_MONITOR_REPORT_PREFIX + "%d"),
```

Comment:
I would hardcode the pixel name and remove the `PixelPrefixes` (as it seems a bit out of place being in the `statistics` module).

```suggestion
    ATP_APP_CPU_MONITOR_REPORT("m_atp_ev_cpu_usage_%d"),
```